Sources/MapboxMaps/Foundation/MapboxMap.swift

```diff
@@ -1008,15 +1008,17 @@ extension MapboxMap {
 
 extension MapboxMap {
     /// Errors related to MapProjection API
+    @available(iOS, obsoleted: 11.0)
     @_spi(Experimental) public enum MapProjectionError: Error {
         case unsupportedProjection
     }
 
     /// Set map projection for Mapbox map.
     /// - Parameter mode: The `MapProjection` to be used by the map.
     /// - Throws: Errors during encoding or `MapProjectionError.unsupportedProjection` if the supplied projection is not compatible with the SDK.
+    @available(*, deprecated, message: "Use `Style.setMapProjection(_:)` instead")
```

Comment:
Would `renamed:` work in this situation?

Reply:
Interesting

> renamed with a keypath to another API; __when provided, Xcode provides an automatic “fix-it”__

This would definitely be useful here

Reply:
Works like a charm - Xcode displays a warning that the method is deprecated and, indeed, provides a "fix-it" 🔥